src/libsync/syncoptions.h

```diff
@@ -43,7 +44,7 @@ class OPENCLOUD_SYNC_EXPORT SyncOptions
     QSharedPointer<Vfs> _vfs;
 
     /** The maximum number of active jobs in parallel  */
-    int _parallelNetworkJobs = 6;
+    std::function<int()> _parallelNetworkJobs = [] { return 6; };
```

Comment:
hm, to make it more easy to understand, i'd add a comment here: // will be overwriten later in loadSyncOptions()